plugins/es-analysis-common/src/test/java/io/crate/analysis/common/CommonAnalyzerITest.java

```diff
@@ -106,4 +108,25 @@ public void testSelectFromRoutines() {
         execute("drop analyzer myanalyzer");
         execute("drop analyzer myotheranalyzer");
     }
+
+    @Test
+    public void testShingleFilterWithGraphOutput() {
+        execute("""
+            CREATE ANALYZER shingle_default (
+                TOKENIZER "standard",
+                TOKEN_FILTERS (
+                    "lowercase",
+                    "stop",
+                    "shingle"
+                )
+            );
+            """);
+        execute("create table t2  (c1 TEXT INDEX USING fulltext WITH (analyzer='shingle_default'));");
+        ensureGreen();
+        execute("insert into t2 (c1) values ('This sentence only has Humans in the text.');");
+        execute("drop analyzer shingle_default");
+
+        var impl = new DisableGraphAttributeImpl();
+        assertThat(impl).isInstanceOf(DisableGraphAttribute.class);
```

Comment:
If we keep a `DisableGraphAttribute` interface - could this instead use `AttributeFactory` to verify that the class can be created dynamically?

Reply:
I only added this bit to stop the implementation being marked as unused (it's created via reflection so static analysis thinks it never gets called) - the actual test for this working is the insert statement above, which will fail if there's no implementation that can be instantiated.

Reply:
Fair point.  I'll open a follow-up to make the test less confusing.